xrpl/deploy-multisig.js

```diff
@@ -0,0 +1,112 @@
+const xrpl = require('xrpl');
+const { Command, Option } = require('commander');
+const { mainProcessor, getWallet, getAccountInfo, sendTransaction, hex } = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const { printInfo, printWarn } = require('../common');
+
+const MAX_TICKET_COUNT = 250;
+const KEY_TYPE = xrpl.ECDSA.secp256k1;
+
+const BASE_RESERVE_XRP = 10;
+const OWNER_RESERVE_XRP = 0.2;
+
+async function processCommand(_, chain, options) {
+    const client = new xrpl.Client(chain.rpc);
+    await client.connect();
+
+    const wallet = await getWallet(chain, options);
+    const balance = Number((await getAccountInfo(client, wallet.address)).Balance) / 1e6;
+    const multisigReserve = BASE_RESERVE_XRP + MAX_TICKET_COUNT * OWNER_RESERVE_XRP;
+    if (balance < Number(multisigReserve)) {
+        printWarn(`Wallet XRP balance is insufficient to fund the multisig account reserve (${multisigReserve} XRP)`);
+        process.exit(0);
+    }
+
+    printInfo('Creating multisig account');
+    const multisig = xrpl.Wallet.generate(KEY_TYPE);
+    printInfo('Created multisig account', multisig.address);
+
+    const paymentTx = {
+        TransactionType: "Payment",
+        Account: wallet.address,
+        Destination: multisig.address,
+        Amount: xrpl.xrpToDrops(multisigReserve),
+    };
+
+    printInfo(`Funding multisig account with ${multisigReserve} XRP from wallet`, JSON.stringify(paymentTx, null, 2));
+    await sendTransaction(client, wallet, paymentTx);
+    printInfo('Funded multisig account');
+
+    const signerListSetTx = {
+        TransactionType: "SignerListSet",
```

Comment:
Looks like this tx allows replacing a normal account to convert it into a multisig? Interesting

Reply:
Yep, that's correct.